packages/firebase/pubspec.yaml

```diff
@@ -0,0 +1,14 @@
+name: firebase
+description: Dart wrapper for native Firebase mobile SDKs
```

Comment:
We might want to say "for mojo" or "for flutter".

There's already two firebase packages in pub. :)